tests/cxx/declare_symbol.h

```diff
@@ -0,0 +1,13 @@
+//===--- declare_symbol.h - test input file for iwyu ----------------------===//
```

Comment:
I'm not sure that works, the files currently in subdirs (IIRC) are just headers that need to be included with a special name. But it might very well work, it hasn't occurred to me before.